warehouse/query-core/src/main/java/datawave/query/composite/CompositeTerm.java

```diff
@@ -85,7 +89,19 @@ public void getNodesAndExpressions(List<Class<? extends JexlNode>> nodeClasses,
      * @return
      */
     private String getAppendedExpressions() {
-        return String.join(separator, expressionList);
+        final List<String> newExpressions;
+        if (Iterables.getLast(jexlNodeList) instanceof ASTERNode && !jexlNodeList.stream().allMatch(x -> x instanceof ASTERNode)) {
+            // We have an ASTERNode in the last position, but the rest are not ERNodes ( and therefore must be EQNodes )
+            // escape all of the terms
+            newExpressions = expressionList.stream().map(Pattern::quote).collect(Collectors.toList());
+            }).collect(Collectors.toList());
```

Comment:
Think this line got left in by mistake. Maybe my suggestion was supposed to remove it or something, but didn't.

Reply:
Yeah. I just force pushed to remove that. it was probably something in the way I accepted the change.